Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -13,6 +13,8 @@
 
 namespace Hazel {
 	
+	float Window::highDPIScaleFactor = 1.0f;
```

Comment:
Basically add a getter and setter function for this? Also would this be better suited to go in ``WindowProps``?

Reply:
What do you mean by WindowProps? Do you mean create a new script? or do you mean a class? I am honestly confused by this...

Reply:
I figured out what you meant, i am sleepy :P I won't do that unless Cherno suggests to do that because it doesn't really add anything to it, also the WindowProps struct is iirc only used in application's window creation, where as the Window class is more used.

Reply:
Ah, i see now what you meant by your questions. I don't like unneeded getters and/or setters, so i won't make them unless Cherno really wants them instead of a public static variable. Also there really is no difference in having the s_HighDPIScaleFactor to be inside WindowProps. But i still want to thank you for your suggestions. 😃